llarp/handlers/tun.cpp

```diff
@@ -934,7 +949,10 @@ namespace llarp
           if (pkt.IsV4())
```

Comment:
This conditional (starting at `if (m_state->m_ExitEnabled)`) feels poorly structured.

Reply:
it may but it's the most readable way to do the logic required.